packages/ckeditor5-widget/src/widgetresize/resizer.ts

```diff
@@ -374,11 +354,7 @@ export default class Resizer extends ObservableMixin() {
 	/**
 	 * Calculates the proposed size as the resize handles are dragged.
 	 *
-	 * @private
-	 * @param {Event} domEventData Event data that caused the size update request. It should be used to calculate the proposed size.
-	 * @returns {Object} return
-	 * @returns {Number} return.width Proposed width.
-	 * @returns {Number} return.height Proposed height.
```

Comment:
Why delete this?

Reply:
This syntax is AFAIR not supported by TypeDoc.